recipe/meta.yaml

```diff
@@ -77,10 +77,12 @@ outputs:
   - name: libllvm{{ major_ver }}
     script: install_llvm.sh                         # [not win]
     build:
-      skip:  true  # [(win and vc<14) or ppc64le]
+      skip:  true  # [win and vc<14]
       activate_in_script: True
       run_exports:                                                 # [not win]
-        - {{ pin_subpackage("libllvm" + major_ver, exact=True) }}  # [not win]
+        - {{ pin_subpackage("libllvm" + major_ver, max_pin="x.x") }}  # [not win]
+        # We should specify build string in the run export
+        # - {{ pin_subpackage("libllvm" + major_ver, exact=True) }}  # [not win]
```

Comment:
Please revert this. `exact=True` adds the build string

Reply:
Yes, I was doing this for testing my hypothesis on the build failure.

It seems that the `run_export` causes conflicts in the test environment.